tests/core/test_utilities.py

```
@@ -641,36 +641,36 @@ class TestChartsUtilsInternalFunctions:
     def test_dec_hour_join(self):
-        from kerykeion.charts.charts_utils import decHourJoin
+        from kerykeion.charts.charts_utils import hms_to_decimal_hours
 
-        assert decHourJoin(12, 30, 0) == pytest.approx(12.5, abs=0.001)
+        assert hms_to_decimal_hours(12, 30, 0) == pytest.approx(12.5, abs=0.001)
 
     def test_offset_to_tz_none_raises(self):
-        from kerykeion.charts.charts_utils import offsetToTz
+        from kerykeion.charts.charts_utils import timedelta_to_decimal_hours
 
         with pytest.raises(Exception):
```

Ruff (0.15.18)

[warning] Do not assert blind exception: `Exception`

(B017)